src/lets_be_rational/normal_distribution.rs

```diff
@@ -1,16 +1,28 @@
-use once_cell::sync::Lazy;
-use statrs::distribution::{ContinuousCDF, Normal};
+use statrs::{consts::SQRT_2PI, function::erf};
+use std::f64::consts::SQRT_2;
 
-static STANDARD_NORMAL: Lazy<Normal> = Lazy::new(|| Normal::new(0.0, 1.0).unwrap());
```

Comment:
are u sure about dropping `statrs` implementation?
any evidence for that?

Reply:
I tried going the other way (having them all use statrs directly) and that slowed down the pricing bench by about 15x on my machine.

Reply:
i think standard_normal - 0 - 1 is quick especially on lazy. can u bring a some perf tests ?

Reply:
Ideally we would be able to call `statrs::distribution::normal::cdf_unchecked`, but it's not part of the public API afaict

Reply:
tests to make evidence for that

Reply:
sure

Reply:
probably it is last functionality from `statrs` 😅

Reply:
Okay, so actually I think the performance blow-up had something weird to do with the static instantiation of `STANDARD_NORMAL`. Having each method call `Normal::standard()` instead of accessing the lazy value avoids the performance blowup I was observing for the pricing benchmark

Reply:
Final performance:
`calc_price` 1.20 -> 1.12 ~ 6%
good stuff :)